java/maven/src/org/netbeans/modules/maven/execute/ReactorChecker.java

```diff
@@ -94,7 +91,7 @@ public class ReactorChecker implements PrerequisitesChecker {
      * @return its apparent reactor root; maybe just the same project
      */
     public static @NonNull NbMavenProject findReactor(@NonNull NbMavenProject module) { // #197232
-        MavenProject prj = module.getMavenProject();
```

Comment:
This is important, as a subproject cannot be typically read with empty Maven repository. But the partial project contains a correct parent reference.